pages/index.js

```diff
@@ -1,95 +1,124 @@
 import Head from "next/head";
-<<<<<<< HEAD
 import { useRouter } from "next/router";
+import { useState } from 'react';
 import { siteTitle } from "../components/Layout";
 import {
   getAuth,
-  connectAuthEmulator,
   signInWithEmailAndPassword,
-  createUserWithEmailAndPassword
+  createUserWithEmailAndPassword,
+  onAuthStateChanged
 } from "firebase/auth"
 import { firebaseApp } from "../config/firebaseConfig";
-import {
-  txtEmail,
-  txtPassword,
-  btnLogin,
-  btnSignup
-} from "../config/authUI";
-import LoginForm from "../components/login/LoginForm";
-import SignUpForm from "../components/login/SignUpForm";
-import { async } from "@firebase/util";
-
-
-const auth = getAuth(firebaseApp);
-connectAuthEmulator(auth, "http://localhost:9099");
 
-//Logging in:
 
-const loginEmailPassword = async () => {
-  const loginEmail = txtEmail.value;
-  const loginPassword = txtPassword.value;
-  try {
-    const userCredential = await signInWithEmailAndPassword(auth, loginEmail, loginPassword);
-    console.log(userCredential.user);
-  }
-  catch (error) {
-    console.log(error);
 
-  }
-}
+const Home = () => {
 
-btnLogin.addEventListener("click", loginEmailPassword);
+  const router = useRouter();
+  const auth = getAuth(firebaseApp);
 
+  const [loginEmail, setLoginEmail] = useState('');
+  const [loginPassword, setLoginPassword] = useState('');
+  const [userName, setUserName] = useState('');
+  const [signupEmail, setSignupEmail] = useState('');
+  const [signupPassword, setSignupPassword] = useState('');
+  const [signupPassConf, setSignupPassConf] = useState('');
+  const [loginNotification, setLoginNotification] = useState('');
+  const [signupNotification, setSignupNotification] = useState('');
 
-//Creating account:
+  onAuthStateChanged(auth, (user) => {
+    if (user) {
+      const uid = user.uid;
+    } else {
 
-const createAccount = async () => {
-  const loginEmail = txtEmail.value;
-  const loginPassword = txtPassword.value;
+    }
+  });
 
-  try {
-    const userCredential = await createUserWithEmailAndPassword(auth, loginEmail, loginPassword);
-    console.log(userCredential.user);
+  const handleLogin = (e) => {
+    e.preventDefault();
+    signInWithEmailAndPassword(auth, loginEmail, loginPassword)
+      .then(
+        setLoginNotification(
```

Comment:
Inside here is where the router.push({}) call needs to be, where the user is successfully signed in.